sematic/cli/BUILD

```diff
@@ -117,6 +117,8 @@ sematic_py_lib(
         ":cli",
```

Comment:
This file change is here for illustrative and reproducibility purposes. It will be removed before merging. Its contents will be included in a dedicated PR that exposes the Build System to users.